lib/template.html

```diff
@@ -97,6 +97,8 @@
   {% assign seo_page_image = seo_page_image | escape %}
 {% endif %}
 
+{% assign seo_page_locale = page.lang | default: site.lang | default: "en_US"" %}
```

Comment:
Yes we can.

The [OG docs](https://developers.facebook.com/docs/reference/opengraph/object-type/website/) default to “en_US”. I thought it made sense to be explicit. If you feel otherwise, we can drop the tag without a defined language (or even if it is “en_US”).